### ISSUE TITLE
Add instructions for WingsX Application

### ISSUE THREAD

**Issue by arian6874** [bootmortis/iran-hosted-domains#48]
I would like to make a feature request to the author. The feature I have in mind is to add an instruction to the wingsX application, which is a great app for iOS and macOS devices. It can be routed to different domains and bypass censorship. I think this feature would be useful

**Comment by Phoenix-999** [comment]
https://github.com/iranxray/hope/issues/90

**Comment by bootmortis** [comment]
@Phoenix-999 thx for your help, I check the iranxray/hope config file and looks like they use old v2fly category name format https://github.com/iranxray/hope/issues/66  
![msedge_a7nB4SC68k](https://github.com/bootmortis/iran-hosted-domains/assets/120013889/4e3b31c7-cbbb-4d51-a352-506082411d39)


So this shouldn't work.

**Comment by Phoenix-999** [comment]
@bootmortis Hope you are doing well
unfortunately that is how this app works, there is no field that we can write and make the input manually when we creating the routing rules, it only has radio button option and dropdown menu that you can choose from. look at the screen shots below for your ref.
nevertheless it seems that it is working correctly so far
The good portion of Iran.dat domain list are now bypassing proxy server and connect directly to the local server.

![Screenshot 2023-05-18 at 20 00 51](https://github.com/bootmortis/iran-hosted-domains/assets/127796122/66767077-7239-4065-af93-a670e66ce593)

**Comment by bootmortis** [comment]
You misunderstood me; there is no category called `category-ir-gov`, check https://github.com/v2fly/domain-list-community/issues/1646
nevertheless, using v2fly ir-related categories is redundant, since `iran.dat` already includes their domains (#13 )

**Comment by Phoenix-999** [comment]
@bootmortis Oh i got you, my bad. thanks for the tip i'll share that with Wings X & FoXray developers I believe they are from chinese XRay community. 
They have a page in telegram (@foxrayapple) and take notes of all users experiences, but they are bit slow in debugging and updating the app frequently. :)

**Comment by Phoenix-999** [comment]
@bootmortis Follow up on my last message, Below is what he says;
![Screenshot 2023-05-19 at 06 54 28](https://github.com/bootmortis/iran-hosted-domains/assets/127796122/04f403aa-f438-4e97-b18a-b98b9d4ba2cc)

**Comment by bootmortis** [comment]
@Phoenix-999 
ممنون ولی ما بنا به دلایلی که بارها در PR های v2fly ذکر کردیم مخالف مرج شدن لیست ما در v2fly هستیم.  
بنظرم مشکل از کلاینت هست که چنین فیچری که اکثر کلاینت‌ها دارن رو ساپورت نمی‌کنه.

**Comment by bootmortis** [comment]
@Phoenix-999 Thank you for notifying them, but we have already stated our stance against integrating our list into v2fly. Additionally, the WingsX client is missing a common feature many other clients have.

**Comment by Phoenix-999** [comment]
I totally agree, I have already made a suggestion to Iranxray team to start developing a client app which is specifically designed for Iranian users with all necessary features and advantages.

No one understands these issues better than those who live in Iran, and wrestling with this level of censorship on a daily basis.

We need to come together and make the last stand for our community or everything that we've been doing, would be for nothing.

Maybe you can get involved in this and make some suggestions.

https://github.com/iranxray/hope/discussions/95

**Comment by yiguous** [comment]
> I totally agree, I have already made a suggestion to Iranxray team to start developing a client app which is specifically designed for Iranian users with all necessary features and advantages.
> 
> No one understands these issues better than those who live in Iran, and wrestling with this level of censorship on a daily basis.
> 
> We need to come together and make the last stand for our community or everything that we've been doing, would be for nothing.
> 
> Maybe you can get involved in this and make some suggestions.
> 
> [iranxray/hope#95](https://github.com/iranxray/hope/discussions/95)

Sorry for replying so late. FoXray 1.5.1 has supported custom geosite.data. You can set a custom geosite.dat url to download it.

This feature could be found by Setting -> Rule Set -> set Source to "custom".

I think this issue could be closed.

**Comment by Phoenix-999** [comment]
@yiguous

I am sure, this topic was never that important for you in the first place! 
unfortunately, despite the full support by Iranian community and many donations which have been made to support your project in past couple of months, 

We saw the unfortunate and disappointing level of prejudice and ignorance by you towards the people of this community which to be honest it is disgraceful for someone whose claims he's part of the Project X community.

I would only hope that you grow up and learn to be a better person through time and I will leave you with this Persian poem by Saadi Shirazi, so that maybe you be humbled by those who you Offended and ignored.

"Human beings are body parts of each other,
In creation they are indeed of one essence.
If a body part is afflicted with pain,
Other body parts uneasy will remain.
If you have no sympathy for human pain,
The name of human you shall not retain."


___________________________________________________

![Screenshot 2023-07-06 at 09 02 10](https://github.com/bootmortis/iran-hosted-domains/assets/127796122/de80e910-8480-4978-a77b-efc4c31643cb)

![Screenshot 2023-07-06 at 08 59 17](https://github.com/bootmortis/iran-hosted-domains/assets/127796122/f1d576f7-879d-4554-acbd-ef0a55e5b2e3)


https://t.me/foxrayapple/8980/9007
不是每个人都开tg会员的。
我基本不看这些伊朗人说什么


https://t.me/foxrayapple/8980/9025
分开也好
文化不同，强行理解也没必要。

**Comment by yiguous** [comment]
I think you misunderstand.

I say "不是每个人都开tg会员的。我基本不看这些伊朗人说什么" means I'm not Telegram Membership, and I can not read Persian. So even I saw these messages, I can not understand what these Persian means. **Translation is easy, but understanding is difficult.** There are two Iranian admins in the FoXray group, so I have no need to deal these things, they will do that.

I say "分开也好。文化不同，强行理解也没必要。", because some Chinese users had conflicts with Iranian users. So dividing them to different groups is a better choice.

**Comment by yiguous** [comment]
Firstly, you mentioned "many donations", I must declare that I never accept private donations and I have not received any. If you are talking about App Store revenue from software sales, sorry, that software does not make much money.

Secondly, I have nothing against Iran, I am neutral on politics.

Finally, I have never claimed that I am part of the Xray community. I do not know where you came to such a conclusion. My words and actions are my own and have nothing to do with the Xray community.

**Comment by yiguous** [comment]
I hope we can stop this topic, this is a technical community, it is completely meaningless to discuss whether a software or a person is good or bad.
I know that on Telegram, there are some Iranian channels doing something under the banner of FoXray, I have no interest in that, anything they do has nothing to do with me.
